spec/acceptance/clearance_installation_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "spec_helper"
+
+describe "Clearance Installation" do
+  around do |example|
+    current_path = File.expand_path("../", __FILE__)
+    Dir.chdir("tmp")
```

Comment:
If example raise an exception, there's a chance that line 11 won't get run, right? How about using the block version of `chdir`?